applications/nrf_desktop/board_configuration.rst

```diff
@@ -87,12 +87,12 @@ nRF52832 Desktop Keyboard (``nrf52kbd``)
       * The board supports ``release`` :ref:`nrf_desktop_bluetooth_guide_fast_pair` configuration (``release_fast_pair`` file suffix).
         The configuration uses the MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``), and supports firmware updates using the :ref:`nrf_desktop_dfu` and the :ref:`nrf_desktop_dfu_mcumgr`.
 
-nRF52840 USB Dongle (``nrf52840dongle/nrf52840``) and nRF52833 USB Dongle (``nrf52833dongle``)
+nRF52840 USB Dongle (``nrf52840dongle/nrf52840/bare``) and nRF52833 USB Dongle (``nrf52833dongle``)
```

Comment:
Is this change only for the nRF desktop application? I could see the "nrf52840dongle/nrf52840" target used in other samples as well.
Also, I think we would require a changelog entry.

Reply:
Yes, this change is only for those, who disable nRF5 SDK bootloader. I was able to find this override only in nRF Desktop applications.